code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -724,17 +724,17 @@
 /datum/ammo/bullet/smartgun/New()
 	..()
 	accurate_range = config.short_shell_range
-	damage = config.low_hit_damage
-	penetration= config.mlow_armor_penetration
+	damage = config.lmed_hit_damage
+	penetration= config.low_armor_penetration
 
 /datum/ammo/bullet/smartgun/lethal
 	flags_ammo_behavior = AMMO_BALLISTIC
 	icon_state 	= "bullet"
 
 /datum/ammo/bullet/smartgun/lethal/New()
 	..()
-	damage = config.lmed_hit_damage
-	penetration= config.low_armor_penetration
+	damage = config.low_hit_damage
+	penetration= config.mlow_armor_penetration
```

Comment:
why not literally just make it have the same stats, does turning IFF off somehow make the bullet weaker

Reply:
this is mainly a encouragement, a few SG bullets on a human will still down em so it works for HVH, but if you turn IFF off on xenos it will do less.